package/gluon-setup-mode/files/lib/gluon/setup-mode/rc.d/S96led

```diff
@@ -16,5 +18,10 @@ start() {
 		status_led="$boot"
 	fi
 
+	custom_led="$(lua -e 'print(require("gluon.setup-mode").get_custom_led() or "")')"
```

Comment:
If we really want to do this in Gluon (which I'm not convinced of - having a failsafe indicator in OpenWrt is always valuable), I'd rather name the function `get_status_led()`.

Reply:
@blocktrron your argument for not adding them upstream could be valuable here.

Reply:
I'm fine with it either way. I would just appreciate a clear direction before I test it hopefully one last time.